Utilities/Mergers/include/Mergers/MergerConfig.h

```diff
@@ -71,6 +71,7 @@ struct MergerConfig {
   ConfigEntry<InputObjectsTimespan> inputObjectTimespan = {InputObjectsTimespan::FullHistory};
   ConfigEntry<MergedObjectTimespan, int> mergedObjectTimespan = {MergedObjectTimespan::FullHistory};
   ConfigEntry<PublicationDecision> publicationDecision = {PublicationDecision::EachNSeconds, 10};
+  ConfigEntry<PublicationDecision, std::vector<std::pair<size_t, size_t>>> publicationDecisionNew = {PublicationDecision::EachNSeconds, {{10, 1}}};
```

Comment:
Do you intend to move `publicationDecisionNew` to `publicationDecision` once we are done with migration on the QC side? It seems like a lot of coordination to do.

It would add some code, which would be anyway dropped later, but if we want a smoother transition, you could try creating a class which has constructors for both single `int` and `std::vector<std::pair<size_t, size_t>>`. Then we could do the transition in 3 PRs, one here, one in QC, one here.

Reply:
fine with me, I have made an extra commit.